airbyte-integrations/connectors/source-mailchimp/integration_tests/Email_activity.json

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
it is not correct format, the stream needs to in catalog, in the list of streams